author_api/serializers.py

```diff
@@ -9,6 +12,22 @@
     FriendRelationship,
     FriendRequest )
 
+class ImageSerializer(serializers.BaseSerializer):
+    def to_representation(self, data):
+        return data.name
+
+    def to_internal_value(self, data):
+        try:
+            extension =  guess_extension(guess_type(data[0:23])[0])
+
+            if extension:
```

Comment:
What happens if there is no extension? Does it throw an exception? If so the "if" is redundant. Either that or we're going to end up in a state where nothing gets returned.